exporters/auto/README.md

```diff
@@ -12,7 +12,7 @@ To leverage the GCP extensions to this, simply add the following dependency:
 <dependency>
   <groupId>com.google.cloud.opentelemetry</groupId>
   <artifactId>exporter-auto</artifactId>
-  <version>0.18.0-alpha</version>
+  <version>0.20.0-alpha</version>
```

Comment:
What happened to 0.19?

Reply:
think we forgot to update the docs for it.